### PR TITLE
Lognormal distribution of delay

### PR DESCRIPTION
**Pull request by podtserkovskiy**
The [Log-normal](https://en.wikipedia.org/wiki/Log-normal_distribution) is a pretty accurate description of a server latency. The Log-normal distribution defines by 2 parameters μ and σ. We will compute these parameters from mean and median of a server response time. These values you can see in your monitoring of the production server.
If need you can adjust response time by min and max parameters.

### CODE REVIEW

**Comment by podtserkovskiy** [comment]
But I can't check the documentation.
```

**Comment by tommysitu** [comment]
This is awesome. I will get back with questions, it’s obviously going to take some time to review... really grateful for the contribution!

File: hoverctl/wrapper/hoverfly.go [hunk 1]
```diff
@@ -363,7 +363,7 @@ func handleResponseError(response *http.Response, errorMessage string) error {
 
 		err := json.Unmarshal(responseError, error)
 		if err != nil {
-			return errors.New(errorMessage + "\n\n" + string(errorMessage))
+			return errors.New(errorMessage + "\n\n" + string(responseError))
```
Comment (tommysitu) [inline hoverctl/wrapper/hoverfly.go]: thank you for these small bug fixes 👍

**Comment by tommysitu** [comment]
Very good PR! I am happy to merge it in once you change the field name. We will check out the doc here, don't worry about it. We want to tell users that the fixed delay and lognormal delay are **additive** when they happened to match the same request.

File: core/handlers/v2/simulation_views.go [hunk 2]
```diff
@@ -119,7 +119,8 @@ func ValidateSimulation(json, schema map[string]interface{}) error {
 }
 
 type GlobalActionsView struct {
-	Delays []v1.ResponseDelayView `json:"delays"`
+	Delays          []v1.ResponseDelayView          `json:"delays"`
+	DelaysLogNormal []v1.ResponseDelayLogNormalView `json:"delays-log-normal"`
```
[the remark this answers is not in the document]
Comment (podtserkovskiy) [reply core/handlers/v2/simulation_views.go]: I've fixed it.

[the remark this answers is not in the document]
Comment (tommysitu) [reply core/handlers/v2/simulation_views.go]: Thank you!

[the remark this answers is not in the document]
Comment (podtserkovskiy) [reply core/handlers/v2/simulation_views.go]: Thank you for the merge!

**Pull request merged**